pulseapi/entries/models.py

```diff
@@ -50,8 +51,8 @@ class Entry(models.Model):
         EmailUser,
         related_name='entries',
     )
-    timestamp = models.DateTimeField(
-        auto_now=True,
+    created = models.DateTimeField(
+        default = timezone.now,
```

Comment:
this one is triggering my "timezones are dangerous" senses - we have no guarantee on which timezone we're deployed on, so we may need to make sure this is always, say, UTC, rather than whatever the local time is in the timezone our slug happens to run that hour.